runtime/include/mli/client_runtime.c

```diff
@@ -17,20 +17,18 @@
  * limitations under the License.
  */
 
-#ifndef CHPL_RUNTIME_ETC_MLI_MLI_CLIENT_RUNTIME_C_
-#define CHPL_RUNTIME_ETC_MLI_MLI_CLIENT_RUNTIME_C_
+#ifndef CHPL_RUNTIME_INCLUDE_MLI_CLIENT_RUNTIME_C_
```

Comment:
Why is this a .c file instead of a .h file?  Or alternately, what is a .c file doing in an include directory?

Reply:
I'm glad you commented on this.

These are three source files containing chunks of code included by the Chapel compiler during compilation of a multi-locale library.

There were two options here originally:

a) Have the runtime compile two separate object files containing the code, then just link them in appropriately.
b) Have the compiler include the source files containing the code during compilation.

I went with the latter since it required less modification of Makefiles, etc.

I can't recall quite why I moved the source files into the `runtime/include` directory, but I would like to move them out again. Michael suggested `runtime/etc`, do you think this is ok?

Reply:
Is this during compilation of the multi-locale library to support linking it?  Or maybe a better way to ask this is: when is this compilation being done?  At runtime build time?  (in which case src is probably the right place)  Or at user program/library build time?  (in which case etc is probably better, though I thjink I might like them to be placed in an etc/src subdir instead of directly in etc.

Reply:
It is at user/program/library build time (during a run of the Chapel compiler, to produce a multi-locale library).

I was thinking of moving them to `runtime/etc/src/...` does this seem acceptable?

Reply:
Yeah, that seems clearest in terms of how and when they hook into the rest of the build, I think.